### ISSUE TITLE
Movement Form Key Input Unresponsive. (Movement)

### ISSUE THREAD

**Issue by TonyDougal** [KhaneTrios/Project-Casius#2]
After I committed the implementation of the Variables.vb source and the new Test Form, input from the Movement Keys appears unresponsive. I'm not certain what could be causing this issue, and I want to try and avoid doing a revert. I'll try and diagnose this the usual way by commenting lines of code that I think are conflicting with the programs logic. I actually have no clue what could be causing this, I just know when it started.

**Comment by TonyDougal** [comment]
Also, hitting the button to transition to the next form just makes the program end. This is only a minor issue and an easy fix though.

**Comment by KhaneTrios** [comment]
me.close()  closes the active form whatever it may be. If you would perhaps specify the exact form you want closed then also open the new form, the program might not end. 

Another possible solution is to say to open the new form, then Application.DoEvents() then close the old form.

We did this simply in Bread Quest with 
![c4cb2ebfeb1f3b7b5b1dcd50c3c8961e](https://cloud.githubusercontent.com/assets/11250819/13548785/592af33e-e2c6-11e5-90cc-6c5bf70e9939.png)

**Comment by TonyDougal** [comment]
The second problem was solved by creating a new main form and making it transition to the form I'm doing the tests on. The first issue was temporarily "solved" by making the button's enabled property false. Unfortunately, I thought that the Variable.vb implementation was causing the issues and I removed that update manually to avoid reverting anyone else's work. I have to fix that. I'll open a new issue about the issue caused by solving that issue.
